test/test_enclave_wrangler.py

```diff
@@ -6,37 +6,125 @@
 """
 import os
 import sys
+import unittest
+from pathlib import Path
+from typing import Dict, List, Union
 
+import pandas as pd
+from requests import Response
 
 TEST_DIR = os.path.dirname(__file__)
-PROJECT_ROOT = os.path.join(TEST_DIR, '..')
+PROJECT_ROOT = Path(TEST_DIR).parent
 # todo: why is this necessary in this case and almost never otherwise?
 # https://stackoverflow.com/questions/33862963/python-cant-find-my-module
-sys.path.insert(0, PROJECT_ROOT)
-from enclave_wrangler.new_enclave_api import JSON_TYPE, upload_concept_set_container, upload_concept_set_version
-from enclave_wrangler.dataset_upload import post_to_enclave_from_3csv
+sys.path.insert(0, str(PROJECT_ROOT))
+from enclave_wrangler.new_enclave_api import JSON_TYPE, make_request, upload_concept_set_container, \
+    upload_concept_set_version
+from enclave_wrangler.dataset_upload import post_to_enclave_from_3csv, upload_new_cset_version_with_concepts
 from enclave_wrangler.config import PALANTIR_ENCLAVE_USER_ID_1
 
 
 TEST_INPUT_DIR = os.path.join(TEST_DIR, 'input', 'test_enclave_wrangler')
 
 
-# todo: Can add this test by composing other non-validation tests (then, remove 'PyUnusedLocal' line)
-# todo: can just not create container for now, and test simply (i) add/delete versions, (ii) add/delete concepts
-# noinspection PyUnusedLocal
-def test_dataset_upload(inpath=os.path.join(TEST_INPUT_DIR, 'test_dataset_upload')):
-    """Test upload of a new dataset"""
-    # TODO: use dataset_upload to upload a palantir-3-file set
-    post_to_enclave_from_3csv(inpath, create_cset_container=False, create_cset_versions=False)
-
-    # TODO: do actual uploads as well; but this requires "teardown" completion first
-    # post_to_enclave(inpath)
-
-    # TODO: teardown
-    #  1. "apiName": "delete-omop-concept-set-version"
-    #  2. there's no endpoint to delete a concept-set-container (i think that's the one). but there is a way to archive
-    #    ...so for now, we can use the following; and Amin might add a way for us to delete a specific concept set with
-    #    ...a single known name, just for us, for our unit tests:
+# TODO use proper teardown() methods
+class TestEnclaveWrangler(unittest.TestCase):
+
+    def test_upload(self):
```

Comment:
@Sigfried Moved `backend/standalone_api_calls/change_concepts.py` here. You can run this file right now, and it's set just to run only this test case.